src/script/components/MessagesList/Message/MemberMessage.tsx

```diff
@@ -82,7 +82,7 @@ const MemberMessage: React.FC<MemberMessageProps> = ({
   const isMemberLeave = message.isMemberLeave();
   const isMemberChange = message.isMemberChange();
 
-  const initShowMore = useDisposableRef(element => {
+  const {ref: initShowMore} = useDisposableRef(element => {
```

Comment:
Actually, I'm wondering if we should not get rid of the disposableRef at all and just add a `click` listener on the parent React element and only react to the event if the target has the right class. WDYT?